NewBlackAuthenticator/Migrations/20211210134252_TwoFARegistrations model added.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace NewBlackAuthenticator.Migrations
+{
+    public partial class TwoFARegistrationsmodeladded : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateTable(
+                name: "TwoFARegistrations",
```

Comment:
Heh, I like how persistent you are with the "TwoFA" because, probably, "2FA" is not a valid way to set up class names in C# 🤓 

Could have gone for `UserAuthenticationData` as name 😁